vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -359,12 +359,40 @@ void LedgerImpl::OnLedgerStateLoaded(
 }
 
 void LedgerImpl::SetConfirmationsWalletInfo(
-    const ledger::WalletInfoProperties& wallet_info) {
-  auto confirmations_wallet_info = GetConfirmationsWalletInfo(wallet_info);
-  DCHECK(confirmations_wallet_info.IsValid());
+    const ledger::WalletInfoProperties& wallet_info_properties) {
+  confirmations::WalletInfo wallet_info;
```

Comment:
nit: Can we move this to line 385, so that we declare it only when we would actually use it

Reply:
Fixed